src/SoftwareVersion.php

```diff
@@ -148,8 +119,25 @@ public function showForm($ID, array $options = [])
             $options['candel'] = false;
         }
 
-        $this->showFormButtons($options);
-
+        $twig_params = [
+            'item' => $this,
+            'softwares_id' => $softwares_id,
+            'params' => $options
+        ];
+        // language=Twig
+        echo TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+            {% extends 'generic_show_form.html.twig' %}
+            {% import 'components/form/fields_macros.html.twig' as fields %}
+
+            {% block form_fields %}
+                {% if item.isNewItem() %}
+                    <input type="hidden" name="softwares_id" value="{{ softwares_id }}">
+                {% endif %}
+                {{ fields.htmlField('', get_item_link('Software', softwares_id), 'Software'|itemtype_name(1)) }}
```

Comment:
`1` is the default value.
```suggestion
                {{ fields.htmlField('', get_item_link('Software', softwares_id), 'Software'|itemtype_name()) }}
```